pkg/identities/service.go

```diff
@@ -0,0 +1,101 @@
+package identities
+
+import (
+	"context"
+	"encoding/json"
+	"io"
+	"net/http"
+
+	"github.com/canonical/identity-platform-admin-ui/internal/logging"
+	"github.com/canonical/identity-platform-admin-ui/internal/monitoring"
+	kClient "github.com/ory/kratos-client-go"
+	"go.opentelemetry.io/otel/trace"
+)
+
+type Service struct {
+	kratos kClient.IdentityApi
```

Comment:
I might be a bit confused. So we don't instantiate and use the client in the `client.go`?

Reply:
Hah so this is a valid point, allegedly we should only use what we need and in our case it s the IdentityApi interface

Why is the `internal/kratos/client.go` returning the whole kratos client (which implements the frontendAPi and so on)?well mainly because of standardization across codebases (login UI does the same) but also because I genuinely don't know if we will need other parts of the kratos client

A TODO would be to strip down what the internal/kratos package does and limit it to what we use it for in the application right now, so basically return the identityAPi only

Reply:
gotcha~ thanks for the clarifications.